tests/resources/Nimbus-Util.robot

```diff
@@ -306,7 +306,7 @@ Create a Simple VC Cluster
     Set Environment Variable  TEST_DATASTORE  datastore1
     Set Environment Variable  TEST_RESOURCE  ${cluster}
     Set Environment Variable  TEST_TIMEOUT  30m
-    [Return]  ${vc-ip}
+    [Return]  ${esx1}  ${esx2}  ${esx3}  ${vc}  ${vc-ip}
```

Comment:
All the other files has a reference to the keyword without assigning any values from the function return. So we are not breaking the method contract. I am guessing we should be good?

Create a Simple VC Cluster  datacenter  cls